javaparser-core/src/main/java/com/github/javaparser/ast/expr/AssignExpr.java

```diff
@@ -78,7 +73,7 @@ public String asString() {
         }
 
         public Optional<BinaryExpr.Operator> toBinaryOperator() {
-            switch (this) {
+            switch(this) {
```

Comment:
I like the space before the parenthesis :)

Reply:
Thank you, it's done with Code Generation (c)